app/components/article-comments.js

```diff
@@ -68,7 +68,23 @@ export default Component.extend(InViewportMixin, {
 
     this._super(...arguments);
 
-    if (page !== null) {
+    const { urlThreadId } = this.articleComments.getUrlThreadParams();
+
+    if (this.isUcp) {
+      // to make sure we won't show cached value we have to fetch these comments on FE
+      this.articleComments
```

Comment:
Moving this here will is a little regression imo.

It'll be called on each page view even if user won't scroll to the comment module/bottom of the page. In `didEnterViewport` it was called only when scrolled to the page bottom which reduced the total amount of calls to the service.

Reply:
I moved it to new `loadCommentCount` method so it will not fire twice anymore. It couldn't sit only in `didEnterViewport` because in some cases it wouldn't fire